CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+DEV
+------------------
+* Remove `Yola.subscribe_to_campaign()` and `Yola.cancel_campaign_subscription`
+  mehtods, they aren't allowed to be used externally.
```

Comment:
Still a typo in `mehtods`.  Sorry, I should have made a list in the previous comment for clarity.